scripts/prune-sstate-cache/prune_cache.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python3
+
+import datetime
+import os
+import sys
+import argparse
+
+def num_bytes(N):
+    suffix = ["B", "KiB", "MiB", "GiB", "TiB", "UWOT_M8?!?"]
+    tmp = float(N)
+    for s in suffix:
+        if tmp / 1024 < 1.0:
+            return tmp, s
+        tmp /= 1024
+    return tmp, "UWOT?"
+
+def main(args):
+    root = os.path.abspath(args.dir)
+    try:
+        days = int(args.time)
+    except ValueError:
+        print("Usage: python %s <dir_base> <num_days_old>" % argv[0])
+        return
+    
+    timelimit = datetime.datetime.now() - datetime.timedelta(days=days)
+
+    objects = [] #(timestamp, path, file_size)
+    for path, dirnames, filenames  in os.walk(root, topdown=True):
```

Comment:
Oof! Two spaces! 
```suggestion
    for path, dirnames, filenames in os.walk(root, topdown=True):
```

Reply:
I will do a full `black` run on the file. There are some trailing spaces and probably other things.

Reply:
Done